gfx/webrender/src/bindings.rs

```diff
@@ -529,3 +530,34 @@ pub extern fn wr_destroy(state:*mut WrState) {
     Box::from_raw(state);
   }
 }
+
+#[no_mangle]
+pub extern fn wr_init() {
```

Comment:
This looks like leftover from rebasing. I renamed wr_init to wr_init_window in the e10s patches.